versions/3.0.md

```diff
@@ -1128,20 +1127,38 @@ A request body with a referenced model definition.
       "schema": {
         "$ref": "#/components/schemas/User"
       },
-      "examples": [ "http://foo.bar/examples/user-example.json" ]
+      "examples": {
```

Comment:
I'm not sure I understand, with `examples` being plural should the value be an array?